test/cache_test.lua

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env tarantool
+
+local t = require('luatest')
+local db = require('test.db')
+local ddl = require('ddl')
+local cache = require('ddl.cache')
+local helper = require('test.helper')
+
+local SPACE_NAME_IDX      = 1
+local SHARD_FUNC_NAME_IDX = 2
+local SHARD_FUNC_BODY_IDX = 3
+
+local primary_index = {
+    type = 'HASH',
+    unique = true,
+    parts = {
+        {path = 'string_nonnull', is_nullable = false, type = 'string'},
+        {path = 'unsigned_nonnull', is_nullable = false, type = 'unsigned'},
+    },
+    name = 'primary'
+}
+
+local bucket_id_idx = {
+    type = 'TREE',
+    unique = false,
+    parts = {{path = 'bucket_id', type = 'unsigned', is_nullable = false}},
+    name = 'bucket_id'
+}
+
+local func_body_first  = 'function() return 42 end'
+local func_body_second = 'function() return 24 end'
```

Comment:
What is a difference for `func_body_first` and `func_body_second` from testing point of view?

Reply:
get different results. so we can be sure that trigger was called and cache is actual

Reply:
(It seems, it is not actual anymore. Resolving the discussion.)